layouts/partials/footer/footer.html

```diff
@@ -13,7 +13,13 @@
     {{ $.Scratch.Set "data" $dot.Page.Site.Data.partials.footer }}
 {{ end }}
 
+<!--- footer from websites-modules --->
 {{ partial "footer.html" . }}
 
 <!--- global modals --->
-{{ partial "global-modals/global-modals.html" . }}
+{{ partialCached "global-modals/global-modals.html" . }}
+
+<!--- integrations carousel modal --->
+{{ if in .Params.categories "marketplace" }}
+{{ partial "integrations-carousel/integrations-carousel-modal.html" . }}
+{{ end }}
```

Comment:
I'm also wondering if we removed the partialcached from footer just so we could add this dynamic piece, should this just live somewhere else instead? or should we keep partialcached on footer and pass the integration variant  or something?

Reply:
Also i don't think we can scope this to just marketplace. There are other integrations that have media e.g 
https://github.com/DataDog/integrations-extras/blob/master/1e/manifest.json#L15
https://app.datadoghq.com/integrations/one-e?search=1e

Reply:
The card said to target marketplace but I just enabled the carousel on al integrations with media: ex: https://docs-staging.datadoghq.com/stefon.simmons/integrations-vid-img-carousel/integrations/launchdarkly/

Re: the partialCached I can add it to the bottom of the baseof. I figured the footer/footer was cached originally because of the global-modal partial

Reply:
1e: https://docs-staging.datadoghq.com/stefon.simmons/integrations-vid-img-carousel/integrations/1e/